sky/job/dashboard/dashboard.py

```diff
@@ -12,36 +12,39 @@
 import flask
 import yaml
 
-import sky
-from sky import spot
+from sky import job
```

Comment:
```suggestion
from sky import job as managed job
```
nit

Reply:
+1, similar feeling. Left a similar comment elsewhere. This particular instance and some other instances seem too generic (e.g., job_utils).